proto/proto-backplane-grpc/build.gradle

```diff
@@ -68,35 +69,40 @@ TaskProvider<Task> generateProtobuf = Docker.registerDockerTask(project, 'genera
   containerOutPath = '/generated'
   imageName = Docker.localImageName('proto-backplane-grpc')
   copyOut {
-    into('build/generated/source/proto/main')
+    into(protoOutputDir)
   }
 }
 
+def syncProtocJava = tasks.register('syncProtocJava', Sync) {
+  dependsOn(generateProtobuf)
+  from(protoOutputDir.get().dir('java')) {
+    from 'java'
+  }
```

Comment:
I'm a bit confused about `from 'java'`... I don't think it does anything? At least removing it, the sync seems to still work...